lib/modules/dosomething/dosomething_rogue/dosomething_rogue.module

```diff
@@ -45,6 +45,23 @@ function dosomething_rogue_client() {
 }
 
 /**
+  * Send a GET request to return all reportback items matching a given
+  * query from Rogue.
+  *
+  * @param array $inputs
+  */
+function dosomething_rogue_get_reportback_items($inputs)
+{
+  $client = new Rogue(ROGUE_API_URL . '/v1/',
```

Comment:
Oh, hmm okay. I don't think we should be including versions in base URL in that case, but can handle that in a different pull request later on.